cron/centAcl-Func.php

```diff
@@ -534,14 +531,14 @@ function getMyHostServicesByName($host_id = null)
  */
 function getMetaServices($resId, $db, $metaObj)
 {
-    $sql = "SELECT meta_id FROM acl_resources_meta_relations WHERE acl_res_id = {$db->escape($resId)}";
+    $sql = "SELECT meta_id FROM acl_resources_meta_relations WHERE acl_res_id = " . $resId;
     $res = $db->query($sql);
     $arr = array();
     if ($res->rowCount()) {
         $hostId = $metaObj->getRealHostId();
         while ($row = $res->fetch()) {
             $svcId = $metaObj->getRealServiceId($row['meta_id']);
-            $arr['_Module_Meta']['meta_' . $row['meta_id']] = "$hostId,$svcId";
+            $arr[$hostId][$svcId] = 1;
```

Comment:
Why you fix the value "1"?

Reply:
I guess cause isset is better (in performance) than in_array